### ISSUE TITLE
OfficeRuntime.auth.getAccessToken hanging after recent update

### ISSUE THREAD

**Issue by alanlemarquand** [OfficeDev/office-js#2152]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

When trying to get a Token during the new email event on Outlook Desktop.  This call
`const token = await OfficeRuntime.auth.getAccessToken({ allowConsentPrompt: false, allowSignInPrompt: false });`
never completes or returns.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC Desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Version 2110 Build 16.0.14524.20000) 64-bit
* Operating System: Windows 10 Enterprise 20H2 
* Browser (if using Office on the web): ______
 
## Expected behavior
A token is returned and was return prior to 1.0.205 of Office.js

## Current behavior
Call to the method hands

## Steps to reproduce

1. Using the manifest below. Open Outlook and go to the **Get Add-in** button in the ribbon
2. Under My Add-ins add the test using the Manifest
3. Open a new email. The body will update with Start and then nothing.

## Link to live example(s)
1. https://alanlemarquand.github.io/ajax-addin-master/manifiest.xml

# Provide additional details

## Context
This is blocking our development and release of the Desktop version of our add-in

## Useful logs

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by exextoc** [comment]
`OfficeRuntime.auth.getAccessToken` and related APIs, are currently blocked for Event-Based Add-ins. See this link: https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/autolaunch#event-based-activation-behavior-and-limitations

We recently unblocked it, however, we haven't rolled it out broadly to everyone yet.
We will update our documentation very soon with information about SSO support for Event-Based add-ins.

**Comment by alanlemarquand** [comment]
Is there any details on when the general rollout will be? We're looking to release the add-in for Outlook Desktop and this is the last part we need.

**Comment by exextoc** [comment]
The fix is rolled out worldwide now.
It is available in Outlook build 16.0.14326.20258 and above.
We are writing up a set of instructions that are required to allow SSO to work in Event-Based Add-ins.
That should be online on our documentation shortly. We will update this thread once it is available.
